src/Table/Snowflake/SnowflakeTableReflection.php

```diff
@@ -30,11 +31,26 @@ final class SnowflakeTableReflection implements TableReflectionInterface
 
     public function __construct(Connection $connection, string $schemaName, string $tableName)
     {
-        $this->isTemporary = strpos($tableName, SnowflakeTableQueryBuilder::TEMP_TABLE_PREFIX) === 0;
-
         $this->tableName = $tableName;
         $this->schemaName = $schemaName;
         $this->connection = $connection;
+
+        $this->setIsTemporary();
+    }
```

Comment:
Zrovna ti napíšu, že toto v konstuktoru nevolaj :) Jestli chceš tak si to ulož do té prop když voláš poprvé isTemporary, ale v konstruktoru určitě nevolaj žadné SQL.

Reply:
jo, jasne, byl jsem tak happy z toho `kind`, ze jsem toto nedomyslel